lib/internal/event_target.js

```diff
@@ -65,7 +65,7 @@ const kWeakHandler = Symbol('kWeak');
 const kResistStopPropagation = Symbol('kResistStopPropagation');
 
 const kHybridDispatch = SymbolFor('nodejs.internal.kHybridDispatch');
-const kRemoveWeakListenerHelper = Symbol('nodejs.internal.removeWeakListenerHelper');
+const kRemoveWeakListenerHelper = Symbol('removeWeakListenerHelper');
```

Comment:
```suggestion
const kRemoveWeakListenerHelper = Symbol('kRemoveWeakListenerHelper');
```

Reply:
Thank you for your feedback. Applied your suggestions.